src/dp.rs

```diff
@@ -19,50 +63,49 @@ pub trait DifferentialPrivacyBudget {}
 /// Marker trait for differential privacy scalar noise distributions.
 pub trait DifferentialPrivacyDistribution {}
 
-/// Zero-concentrated differential privacy (zCDP) budget as defined in [[BS16]].
+/// Zero-concentrated differential privacy (ZCDP) budget as defined in [[BS16]].
 ///
 /// [BS16]: https://arxiv.org/pdf/1605.02065.pdf
 pub struct ZeroConcentratedDifferentialPrivacyBudget {
-    /// Parameter `epsilon`, using the notation from [[CKS20]] where `rho = (epsilon**2)/2`
-    /// for a `rho`-zCDP budget.
+    epsilon: Ratio<BigUint>,
+}
+
+/// Alias for ZeroConcentratedDifferentialPrivacyBudget.
+pub type ZCdpBudget = ZeroConcentratedDifferentialPrivacyBudget;
+
+impl ZCdpBudget {
+    /// Create a budget for parameter `epsilon`, using the notation from [[CKS20]] where `rho = (epsilon**2)/2`
+    /// for a `rho`-ZCDP budget.
     ///
     /// [CKS20]: https://arxiv.org/pdf/2004.00010.pdf
-    pub epsilon: Rational,
+    pub fn new(epsilon: Rational) -> Self {
+        ZCdpBudget { epsilon: epsilon.0 }
+    }
 }
 
 impl DifferentialPrivacyBudget for ZeroConcentratedDifferentialPrivacyBudget {}
 
-/// Zero-mean Discrete Gaussian noise distribution.
-///
-/// The distribution is defined over the integers, represented by arbitrary-precision integers.
-#[derive(Clone, Debug)]
-pub struct DiscreteGaussian {
-    /// Standard deviation of the distribution.
-    pub sigma: Rational,
-}
-
-impl DifferentialPrivacyDistribution for DiscreteGaussian {}
+/// Strategy to make aggregate results differentially private, e.g. by adding noise from a specific
+/// type of distribution instantiated with a given DP budget.
+pub trait DifferentialPrivacyStrategy {
+    /// The type of the DP budget, i.e. the variant of differential privacy that can be obtained
+    /// by using this strategy.
+    type Budget: DifferentialPrivacyBudget;
```

Comment:
nit: For consistency with other traits in this repo, add a blank line after associated type declarations. (Here and below.)
```suggestion
    type Budget: DifferentialPrivacyBudget;
    
```

Reply:
Done.